File: javascript/3465.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var [a, b, c] = input.split(' ').map(e => parseInt(e))

semiperimetro = (a + b + c) / 2;
area = Math.sqrt(semiperimetro * (semiperimetro - a) * (semiperimetro - b) * (semiperimetro - c))
console.log(area.toFixed(3) + " m2")

File: C/3465.c
#include <stdio.h>
#include <math.h>

int main(){
    int a, b, c;
    double area, semiperimetro;
    scanf("%d %d %d", &a, &b, &c);
    semiperimetro = (a + b + c) / 2.0;
    
    area = sqrt(semiperimetro * (semiperimetro - a) * (semiperimetro - b) * (semiperimetro - c));
    printf("%.3lf m2\n", area);

    return 0;
}